SwiftTerm/Sources/SwiftTerm/Mac/MacTerminalView.swift

```diff
@@ -115,6 +114,9 @@ public class TerminalView: NSView, NSTextInputClient, NSUserInterfaceValidations
         
     func setup(frame: CGRect, bounds: CGRect)
     {
+        wantsLayer = true
```

Comment:
What do we use the layer for?   Is this a good idea?

Reply:
This is a good question.

I used it to have a background without drawing because when drawing in context, it interferes with how selection/background color blending.

I didn't notice a side effect, but now I'm not sure

on one hand:

https://developer.apple.com/documentation/appkit/nsview/1483695-wantslayer
> In a layer-backed view, any drawing done by the view is cached to the underlying layer object. This cached content can then be manipulated in ways that are more performant than redrawing the view contents explicitly. 

on the other

https://developer.apple.com/documentation/appkit/nsview/1483686-drawrect
> If your app manages content using its layer object instead, use the updateLayer method to update your layer instead of overriding this method.

Reply:
I think we're good, after checking wantsUpdateLayer()

https://developer.apple.com/documentation/appkit/nsview/1483461-wantsupdatelayer

> A view can update its contents using one of two techniques. It can draw those contents using its draw(_:) method or it can modify its underlying layer object directly. During the view update cycle, each dirty view calls this method on itself to determine which technique to use. The default implementation of this method returns false, which causes the view to use its draw(_:) method.